api/paidAction/transfer.js

```diff
@@ -0,0 +1,57 @@
+import { satsToMsats } from '@/lib/format'
+import { notifyDeposit } from '@/lib/webPush'
+import { PAID_ACTION_PAYMENT_METHODS } from '@/lib/constants'
+
+export const anonable = true
+
+export const paymentMethods = [
+  PAID_ACTION_PAYMENT_METHODS.P2P
+]
+
+export async function getCost ({ sats }) {
+  return satsToMsats(sats)
+}
+
+export async function invoiceablePeer ({ targetUserId }, { models }) {
+  return targetUserId
+}
+
+export async function perform ({ invoiceId, sats, description, descriptionHash, comment, targetUserId }, { me, tx }) {
+  const invoice = await tx.invoice.update({
+    where: { id: invoiceId },
+    data: {
+      comment
+    }
+  })
+  await notifyDeposit(targetUserId, invoice)
+  return { sats, targetUserId }
+}
+
+export async function describe ({ sats, description, descriptionHash }, context) {
+  return `SN: ${description ?? ''}`
+}
+
+export async function describeHash ({ sats, description, descriptionHash }, context) {
```

Comment:
It looks like you added quite a few new exports from `paidAction` modules. Can you document those in the README?

Reply:
since these are fields introduced by the other pr i will just leave them out for now